mmv1/products/apigee/Organization.yaml

```diff
@@ -82,47 +82,49 @@ examples:
       - properties
     skip_docs:
       true
-      # Resource creation race
+    # Resource creation race
     skip_vcr: true
   - !ruby/object:Provider::Terraform::Examples
     name: 'apigee_organization_cloud_full'
     skip_test:
       true
-      # This is a more verbose version of the above that creates all
-      # the resources needed for the acceptance test. While all Apigee
-      # resources in this test are in the GA API, we depend on a service
-      # identity resource which is only available in the beta provider.
+  # This is a more verbose version of the above that creates all
+  # the resources needed for the acceptance test. While all Apigee
+  # resources in this test are in the GA API, we depend on a service
+  # identity resource which is only available in the beta provider.
   - !ruby/object:Provider::Terraform::Examples
     name: 'apigee_organization_cloud_full_test'
     primary_resource_id: 'org'
     test_env_vars:
       org_id: :ORG_ID
       billing_account: :BILLING_ACCT
+    ignore_read_extra:
+      - properties
```

Comment:
Is it still necessary to have `properties` included in `ImportStateVerifyIgnore` for acceptance tests? I assume the flatten function will perform the necessary reordering during the import test step, but it could be my mental model of the test framework is incorrect.

Reply:
I tried taking these out and [one of the tests](https://hashicorp.teamcity.com/buildConfiguration/TerraformProviders_GoogleCloud_GOOGLE_BETA_MMUPSTREAMTESTS_GOOGLEBETA_PACKAGE_APIGEE/132406?buildTab=tests&expandedTest=build%3A%28id%3A132406%29%2Cid%3A2000000000) failed at the ImportStateVerify step. I believe that is expected (?) because import logic doesn't have a config to match the ordering of, so it'll always end up being in alphabetical order. The ordering logic should still be getting verified by the check for a diff after apply.

Reply:
That makes sense - my mental model was that import results in the Import function running and then the Read function running (so `flattenApigeeOrganizationPropertiesProperty` would be used), but even if there's a resource block containing an ordered list that's present at the time of import that information isn't necessarily available during that operation.

Thanks for checking!